sdk/src/main/scala/com/horizen/storage/SidechainStateStorage.scala

```diff
@@ -64,23 +72,32 @@ class SidechainStateStorage (storage : Storage, sidechainBoxesCompanion: Sidecha
     }
   }
 
-  def getWithdrawalRequests(epoch: Int) : JList[WithdrawalRequestBox] = {
+  def getWithdrawalRequests(epoch: Int) : Seq[WithdrawalRequestBox] = {
```

Comment:
use this, return option.
getUnprocessedWithdrawalRequests can be removed.

Reply:
I left this method for flexibility only.